js/w3c/fixup.js

```diff
@@ -0,0 +1,40 @@
+
+// Module w3c/fixup
+// Inserts a link to the appropriate JS Extension for the W3C Spec Style Sheet.
+// CONFIGURATION
+//  - specStatus: the short code for the specification's maturity level or type (required)
+
+define(
+    ["core/utils"],
+    function (utils) {
+        return {
+            run:    function (conf, doc, cb, msg) {
+                msg.pub("start", "w3c/fixup");
+                if (!conf.specStatus) msg.pub("error", "Configuration 'specStatus' is not set, required for w3c/fixup");
+                var statStyle = conf.specStatus;
+                if (statStyle === "FPWD"    ||
+                    statStyle === "LC"      ||
+                    statStyle === "WD-NOTE" ||
+                    statStyle === "LC-NOTE") statStyle = "WD";
+                if (statStyle === "FPWD-NOTE") statStyle = "WG-NOTE";
+                if (statStyle === "finding" || statStyle === "draft-finding") statStyle = "base";
+                var fixup = "null";
```

Comment:
Note, there is a bug here. it is set to the string "null", but later you check if it's `null`.